src/components/pages/AddAvatarForm/AddAvatarContainer.js

```diff
@@ -4,7 +4,15 @@ import { useOktaAuth } from '@okta/okta-react';
 import RenderAddAvatar from './RenderAddAvatar';
 
 const AddAvatarContainer = ({ LoadingComponent }) => {
-  const { authState, authService } = useOktaAuth();
+  // const { authState, authService } = useOktaAuth();
+  // augment "oktaAuth" to behave like "authService"
+  const { authState, oktaAuth } = useOktaAuth();
+  oktaAuth.getUser = oktaAuth.token.getUserInfo;
+  oktaAuth.logout = oktaAuth.signOut;
+  oktaAuth.isAuthenticated = authState.isAuthenticated;
+  const authService = oktaAuth;
```

Comment:
That's a very neat way of handling a lot of duplicate initialization code. 👍🏽 I think it would shrink the PR down by half the LOC (lines of code), which is great.

Reply:
Let's not write hooks that we're not going to use